pkgs/tools/nix/nix-output-monitor/default.nix

```diff
@@ -25,6 +26,13 @@ mkDerivation {
     ansi-terminal async attoparsec base containers directory HUnit mtl
     nix-derivation process relude stm text time unix
   ];
+  postInstall = ''
+    cat > $out/bin/nom-build << EOF
+    #!${runtimeShell}
+    ${expect}/bin/unbuffer nix-build "\$@" 2>&1 | exec $out/bin/nom
```

Comment:
```suggestion
    ${expect}/bin/unbuffer nix-build "\$@" |& exec $out/bin/nom
```

Reply:
Can you explain, why you want this changed?

Reply:
That's apparently not supported when calling bash as sh.